ledger/catchpointtracker_test.go

```diff
@@ -269,8 +269,8 @@ func getNumberOfCatchpointFilesInDir(catchpointDir string) (int, error) {
 
 func calculateStateProofVerificationHash(t *testing.T, ml *mockLedgerForTracker) crypto.Digest {
 	var digest crypto.Digest
-	err := ml.dbs.Rdb.AtomicContext(context.Background(), func(dbCtx context.Context, tx *sql.Tx) (err error) {
-		rawData, err := store.GetAllStateProofVerificationContexts(dbCtx, tx)
+	err := ml.dbs.Snapshot(func(dbCtx context.Context, tx *sql.Tx) (err error) {
```

Comment:
Snapshot -> SnapshotContext

Reply:
looks like `Snapshot ` calls `SnapshotContext ` with `context.Background()`. In this test, we don't have an open context, so why shouldn't we just use `Snapshot `?

Reply:
fair